src/extension/netcdf-out/src/main/java/org/geoserver/wcs/responses/NetCDFOutputManager.java

```diff
@@ -1085,6 +1089,24 @@ public void write() throws IOException, InvalidRangeException {
      * Add global attributes to the Dataset if needed
      */
     private void initializeGlobalAttributes() {
+        // Copy global attributes from source NetCDF
```

Comment:
Good. Putting the logic here allows later overwrite by more specific configuration, the same as for variable attributes. I like this.